openapi/resource-server.yaml

```diff
@@ -54,6 +54,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/wallet-address'
+                additionalProperties: true
```

Comment:
Does this override the `additionalProperties: false` on the $ref: https://github.com/interledger/open-payments/blob/00d479191ca7b8e277b36dc6bec480f5d31a19ef/openapi/resource-server.yaml#L796-L800

Reply:
yeah, `additionalProperties: true` on the wallet-address itself is ok IMO